tests/e2e/payloads/accreditation/accredit-jwt.json

```diff
@@ -0,0 +1,14 @@
+{
+	"issuerDid": "did:cheqd:testnet:15b74787-6e48-4fd5-8020-eab24e990578",
+	"subjectDid": "did:cheqd:testnet:BjS4Nv8bVdxm2WW28MCfXA",
+	"schemas": [
+		{
+			"url": "https://schema.org/Person",
+			"type": "Person"
+		}
+	],
+	"format": "jwt",
+	"accreditationName": "accreditAccreditation",
+	"parentAccreditation": "did:cheqd:testnet:5RpEg66jhhbmASWPXJRWrA?resourceName=authorizeAccreditation&resourceType=VerifiableAuthorisationForTrustChain",
+	"rootAuthorization": "did:cheqd:testnet:5RpEg66jhhbmASWPXJRWrA?resourceName=authorizeAccreditation&resourceType=VerifiableAuthorisationForTrustChain"
```

Comment:
Seems like same DID as parent and root. Can we do at least two different levels?

Reply:
@ankurdotb it is done it the attestation test. At one point there should be a single level when the parent is root

Reply:
Added child accreditations